src/schemas/argdown.ts

```diff
@@ -0,0 +1,26 @@
+export type RelationType =
```

Comment:
Why do we need the schema here? I have not found a reference in the rest of the code. It would only be relevant if we used the json export feature of argdown.